pavelib/prereqs.py

```diff
@@ -300,3 +300,9 @@ def install_prereqs():
 
     install_node_prereqs()
     install_python_prereqs()
+    log_installed_python_prereqs()
+
+def log_installed_python_prereqs():
+    """  Logs output of pip freeze for debugging. """
+    sh("pip freeze > {}".format("test_root/log/pip_freeze.log"))
```

Comment:
You want to use a var for `test_root/log`. I think we have that declared somewhere.